silkshopy-ecommerce/src/App.js

```diff
@@ -1,15 +1,28 @@
 import "./App.css";
 import logo from "./logo.png";
 import {products} from "./backend/db/products"
-import { Navbar } from "./components/Navbar/Navbar";
-import { Footer } from "./components/Footer/Footer";
 import { Home } from "./pages/Homepage/Home";
+import { Navbar } from "./components/Navbar/Navbar";
+import { Routes, Route, Link } from "react-router-dom";
+import {ProductListing} from "../src/pages/ProductListing/ProductListing";
+import {Cart} from "../src/pages/Cart/Cart";
+import {Wishlist} from "../src/pages/Wishlist/Wishlist";
+import {Login} from "../src/pages/Authentication/Login";
+import {Signup} from "../src/pages/Authentication/Signup";
```

Comment:
Just a nit pick, you have your destructuring properly indented as some place and at some place it is not, so you can work upon that and indent properly and have proper spaces in all the places.

Reply:
yes i will do it.